services/galley/src/Galley/API/LegalHold.hs

```diff
@@ -89,9 +90,7 @@ removeSettings (zusr ::: tid ::: _) = do
 
 -- | Request to provision a device on the legal hold service for a user
 getUserStatus :: UserId ::: TeamId ::: UserId ::: JSON -> Galley Response
-getUserStatus (zusr ::: tid ::: uid ::: _) = do
-    membs <- Data.teamMembers tid
-    void $ permissionCheck zusr ChangeLegalHoldUserSettings membs
+getUserStatus (_zusr ::: tid ::: uid ::: _) = do
```

Comment:
ah, now i remember that we have a hidden permission bit for specifically this end-point: `ViewLegalHoldUserSettings`.  now i think we should check that here, and just give it to everybody in the default hidden permissions.  then it's easy to change later.